src/Nethermind/Nethermind.Evm/CodeInfoRepository.cs

```diff
@@ -113,9 +118,9 @@ private static ICodeInfo InternalGetCachedCode(IReadOnlyStateProvider worldState
 
         [DoesNotReturn]
         [StackTraceHidden]
-        static void MissingCode(Address codeSource, in ValueHash256 codeHash)
+        static void MissingCode(in ValueHash256 codeHash)
         {
-            throw new NullReferenceException($"Code {codeHash} missing in the state for address {codeSource}");
+            throw new NullReferenceException($"Code {codeHash} missing in the state");
```

Comment:
Can we change the exception? It is confusing to throw `NullReferenceException` explicitly

Reply:
Throwing better exception